java/fury-core/src/main/java/org/apache/fury/AbstractThreadSafeFury.java

```
@@ -0,0 +1,33 @@
+package org.apache.fury;
```

Please properly add license header or include necessary info if it's copied.

---

header added.  Thanks for this catch, I always forget add header for new file.

---

I don't look into details but CI should fail if license headers missing.

---

* the code style check was failing until the header was added
* the Graal VM CI is failing still

---

CI succeed now